app/models/card.rb

```diff
@@ -24,6 +24,12 @@ def handle_check(your_translate)
     self.save!
   end
 
+  def self.notify_cards
+    Card.where("review_date <= ?", Date.today).each do |card|
```

Comment:
Do not use `Date.today` without zone. Use `Time.zone.today` instead.<br>Use `find_each` instead of `each`.